web/styles/settings.css

```diff
@@ -1995,3 +1999,29 @@ $option_title_width: 180px;
             0 0 8px hsl(206deg 80% 62% / 60%);
     }
 }
+
+#generate-integration-url-modal {
+    .inline {
+        display: inline;
+    }
+
+    .integration-url {
+        font-family: "Source Code Pro", monospace;
+        padding: 10px;
+        font-size: 0.85rem;
+        border: 1px solid hsl(0deg 0% 87%);
+        border-radius: 4px;
+        overflow-wrap: break-word;
+        word-break: break-all;
+    }
+
+    #integration-url-selected-stream {
```

Comment:
I don't think this class is used anywhere now, but I guess CSS will still be needed for long stream names.

Reply:
This is how it looks for long stream names which I think is fine. 

![image](https://github.com/zulip/zulip/assets/35286603/3134ebeb-ae55-4866-aaa0-56da99788e4b)

Reply:
The other dropdowns also look like this, but I think keeping the height of button same and change the text to something like `very long very long ......` would be better. We can use tooltips to show the full name. I think we can discuss this in #design on CZO.